### PR TITLE
Release/v0.5.0

### LINKED ISSUE #226: Release v0.5.0
## Release

Release v0.5.0

## Checklist

- [x] Update metadata & clusterctl-settings
- [x] Update docs, (compatibility table, usage etc) .
- [ ] Create tag.
- [ ] Update the created draft release to include (BREAKING Changes, Important Notes).
- [ ] Publish the release.

### PR DESCRIPTION
**Pull request by wikkyk**
fix #226

### CODE REVIEW

**Comment by wikkyk** [comment]
Still waiting on #524

**Comment by mcbenjemaa** [comment]
I have triggered the e2e tests

File: README.md [hunk 1]
```diff
@@ -16,12 +16,13 @@ Check out the [quickstart guide](./docs/Usage.md#quick-start) for launching a cl
...
-| CAPMOX v1alpha1 (v0.3) |             ☓              |             ✓              |             ✓              |
-| CAPMOX v1alpha1 (v0.4) |             ☓              |             ✓              |             ✓              |
+|                        | Cluster API v1beta1 (v1.4) | Cluster API v1beta1 (v1.5) | Cluster API v1beta1 (v1.6) | Cluster API v1beta1 (v1.7) |
+|------------------------|:--------------------------:|:--------------------------:|:--------------------------:|:--------------------------:|
+| CAPMOX v1alpha1 (v0.1) |             ✓              |             ✓              |             ☓              |             ☓              |
+| CAPMOX v1alpha1 (v0.2) |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPMOX v1alpha1 (v0.3) |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPMOX v1alpha1 (v0.4) |             ☓              |             ✓              |             ✓              |             ☓              |
```
[the remark this answers is not in the document]
Comment (wikkyk) [reply README.md:24]: We could but I'd just leave it alone. We haven't tested them, go.mod doesn't point at it.

**Comment by wikkyk** [comment]
e2e has already been run.

**Review by mcbenjemaa — APPROVED** [review]
LGTM

**Pull request merged**